views/weather_api.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record model="ir.config_parameter" id="ir_config_parameter_weather_api_key">
+            <field name="key">flower_shop.weather_api_key</field>
+            <field name="value">unset</field>
```

Comment:
your api key should be here

Reply:
we set it here at first , but we saw this sentence 
"Since the API key will be different and specific for each database, we must never set our API key in the XML record. Instead, we will set it via the user interface/frontend."
so we left it to the user to input it.
anyway , we will update the code and set the value directly